tests/test_discovery_sample/tests_sample.py

```diff
@@ -29,6 +30,18 @@ class EmptyTestCase(TestCase):
     pass
 
 
+class TaggedTestCase(TestCase):
```

Comment:
Need to also test tagging of an entire `TestCase` (and how tags on a case and on individual methods interact with each other).